installscripts/jazz-terraform-unix-noinstances/scripts/configure-gitlab.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```

Comment:
Yeah, this bash will get invoked in terraform external (https://www.terraform.io/docs/providers/external/data_source.html) command, Since it was a try, so used bash as explained in the link.